### ISSUE TITLE
PMS 1.0.0.2261

### ISSUE THREAD

**Issue by Unr3al2** [realriot/tvheadend-ng.bundle#50]
Hey,
Streaming does not work on the new version 1.0.0.2261.

**Comment by lusiada** [comment]
Hello,

I have the same problem, after upgrading Plex Media Server to 1.0.0.2261 streaming stopped working.
I can list the channels and epg but when I start the channel stream it keeps loading for ever (on the webplayer) and on my Samsung smart TV it ends after a while with the message cant find the stream.

**Comment by bmentink** [comment]
Same problem here .... I can't stream the channels either in Plex Media Player or the Web client.
Please fix ASAP, a lot of folk use this plugin ..

**Comment by unheilig** [comment]
same here, ok for channel and EPG listing but streaming of video content does result in a network error or keeps loading forever

**Comment by andreimusca** [comment]
I can confirm that with the latest Plex Media Server is not working. Probably they changed something in the API.
It works though with a media server lower than 0.9.16.0

**Comment by bmentink** [comment]
Hi All,
I can confirm that this plugin DOES work if you change your tvheadend user to use the Matroska stream .. SD works fine, but HD on my system pauses/buffer's .. but that may be another issue.

EDIT: I am using Version 1.0.0.2261 of the server, latest git pull of the plugin.

**Comment by lusiada** [comment]
@bmentink,

I have setup an user with the matroska stream but still no luck.

**Comment by bmentink** [comment]
Did you restart the server and client?
EDIT: WHat client are you using? I am using PMP compiled for Linux. (Version 1.1.2-1)

**Comment by lusiada** [comment]
Yes, no change. Are you using security and password with the TVHEADEND server?

**Comment by bmentink** [comment]
I am using a password on my user, I am not sure what you mean by security, havn't set up anything special ..

**Comment by unheilig** [comment]
I've set up matroska as stream profile, stopped/restarted server and client, but no luck here.
@lusiada: do you mean it's needed an account with password or not?

**Comment by lusiada** [comment]
Hi, No I was asking, some plugins do not work if you have security setup (user with password) but that is not the issue here.
I tried everything, is just not working, also, not streaming HD is a big no go for me.
Is just sad that the developer abandoned the plugin, he should just officially abandon it and leave it for someone to pick the work up where he has stopped.

Now there is just no reaction or any support, sad but this is the risk with free unsupported plugins.

**Comment by unheilig** [comment]
i've downgraded the PMS to a 0.9.15 version...

**Comment by jaarsa** [comment]
PMS version 1.0.0.2261
TH version 4.1-2119........trusty

Prerequisite : 
The user created for PMS in TH needs a proper username and password, including admin rights.

## A working solution.(See pics.)

TVHeadend.png
Create a new streaming profile under (Stream Profiles)
- You can either use original MPEG-TS/av-lib or the MATROSKA/av-lib as both work
- It´s most important that you use AAC in the Audio Codec. Everything else will fail.
  ![tvheadend](https://cloud.githubusercontent.com/assets/7510420/16598895/50779978-4300-11e6-8c4a-8e6f5ffee43c.PNG)

PMS_settings.png
- Disable the MPEG-TS Pass-Through, or else both PMS and TH will try to transcode.
- Enter correct name of the streaming profile you created in TH.
  <img width="599" alt="pms_settings" src="https://cloud.githubusercontent.com/assets/7510420/16598942/7ca67848-4300-11e6-9288-6b038f4d6294.png">

**Comment by bmentink** [comment]
Where is the PMS settings for MPEG-TS pass through .... can't find it ..

On Wed, Jul 6, 2016 at 8:44 AM, jaarsa notifications@github.com wrote:

> PMS version 1.0.0.2261
> TH version 4.1-2119........trusty
> 
> Prerequisite :
> The user created for PMS in TH needs a proper username and password,
> including admin rights.
> A working solution.(See pics.)
> 
> TVHeadend.png
> Create a new streaming profile under (Stream Profiles)
> - You can either use original MPEG-TS/av-lib or the MATROSKA/av-lib as
>   both work
> - It´s most important that you use AAC in the Audio Codec. Everything
>   else will fail. [image: tvheadend]
>   https://cloud.githubusercontent.com/assets/7510420/16598895/50779978-4300-11e6-8c4a-8e6f5ffee43c.PNG
> 
> PMS_settings.png
> - Disable the MPEG-TS Pass-Through, or else both PMS and TH will try
>   to transcode.
> - Enter correct name of the streaming profile you created in TH. [image:
>   pms_settings]
>   https://cloud.githubusercontent.com/assets/7510420/16598942/7ca67848-4300-11e6-9288-6b038f4d6294.png
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/realriot/tvheadend-ng.bundle/issues/50#issuecomment-230597223,
> or mute the thread
> https://github.com/notifications/unsubscribe/AJp6h1WXgKAE3Ub-XPABQRvN4rSonsh3ks5qSsIagaJpZM4I-3KB
> .

## 

---

## Life is a sexually transmitted disease with a 100% mortality rate.

**Comment by jaarsa** [comment]
On the latest version of tvheadend-ng.bundle you will find MPEG-TS Pass-Through settings in the TV-Headend ("Channels" in the web of PMS) 
Hit the settings option on the TH channel.
<img width="768" alt="channels" src="https://cloud.githubusercontent.com/assets/7510420/16600481/98d96208-4307-11e6-8928-49b54541be95.png">

**Comment by lusiada** [comment]
@jaarsa,

Thanks for the howto! I got SD channels working with your help.
Sadly HD is not working, get the error can't play this stream.

SD is working.

**Comment by relkai** [comment]
I can also confirm that even using the built-in matroska profile (no transcoding on the TVH side, therefore no double-transcoding of TVH and PMS) for liveTV and recordings the plugin is still working. Streaming within the local network or transcoding to any device including iOS just works out of the box. I have the latest stable TVH version installed (4.0.9) and the latest PMS (1.0.0.2261). Nearly all of my channels are full HD ones.

**Comment by unheilig** [comment]
good news so far. 
@jaarsa: following your configuration, on my iOS app as client, PMS 1.0.0.2261 and Tvheadend 4.1.2116, plugin works fine. 
tonight i will test on Chrome (win 7) Samsung smart TV and XBOX client, then let you all know.

thank you

**Comment by Unr3al2** [comment]
What you put plex server ?? Windows, Linux?

**Comment by jaarsa** [comment]
UBUNTU 14.04 LTS

**Comment by jaarsa** [comment]
@relkai I can not make the matroska work, without transcoding.
How is your settings in tvheadend.bundle (can you make pics), and how is your player settings in PMS?
I asume you use the default matroska settings in TH?

**Comment by unheilig** [comment]
@Unr3al2 it's running on a x86 synology (avoton)

**Comment by relkai** [comment]
I'm running Gentoo Linux, but Plex-wise this shouldn't be any different than Ubuntu.

@jaarsa Take a look at this thread in the Plex forums with detailed screenshots of my settings:
[https://forums.plex.tv/discussion/comment/1210542/#Comment_1210542](https://forums.plex.tv/discussion/comment/1210542/#Comment_1210542)

And here are my player settings (web player):

![image](https://cloud.githubusercontent.com/assets/20300700/16620360/73485c70-4392-11e6-8c0c-98126a3f57dd.png)
